tests/test_utils.py

```diff
@@ -298,11 +306,536 @@ def check_multistream_classic(fnames):
+                           'datasets',
+                           'ec_flux_towers',
+                           'obs')
+
+    for subdir in os.listdir(obs_dir):
+
+        if check_site_files(obs_dir, subdir):
+            # get CSV file from site directory to get lat/lng for site
+            lat, lng = get_fluxnet_lat_lon(obs_dir, subdir)
+
+            # loop over data to compare
+            data = {}
+            for key, items in compare_data_dict.items():
+
+                if key == "ecflux":
+                    try:
+                        # load Ameriflux data
+                        data[key] = read_fluxnet_obs(subdir,
+                                                     science_test_data_dir,
+                                                     items)
+                    except OSError:
+                        warnings.warn("this site does not have data")
+
+                elif key == "VIC.4.2.d":
+                    try:
+                        # load VIC 4.2 simulations
+                        data[key] = read_vic_42_output(lat, lng,
+                                                       science_test_data_dir,
+                                                       items)
+
+                    except OSError:
+                        warnings.warn("this site has a " +
+                                      "lat/lng precision issue")
+
+                else:
+                    try:
+                        # load VIC 5 simulations
+                        data[key] = read_vic_5_output(lat, lng,
+                                                      science_test_data_dir,
+                                                      items)
+                    except OSError:
+                        warnings.warn("this site has a " +
+                                      "lat/lng precision issue")
+
+            # make figures
+            vars = {'OUT_LATENT': 'LH', 'OUT_SENSIBLE': 'H', 'OUT_SWNET':
+                    'SW_NET', 'OUT_LWNET': 'LW NET'}
+
+            # plot preferences
+            fs = 15
+            dpi = 150
+
+            if 'annual_mean_diurnal_cycle' in plots_to_make:
+
+                # make annual mean diurnal cycle plots
+                with plt.rc_context(dict(sns.axes_style("whitegrid"),
+                                    **sns.plotting_context("notebook"))):
+                    f, axarr = plt.subplots(4, 1, figsize=(8, 8), sharex=True)
+
+                    for i, (vic_var, variable_name) in enumerate(vars.items()):
+
+                        # calculate annual mean diurnal cycle for each
+                        # DataFrame
+                        annual_mean = {}
+                        for key, df in data.items():
+                            annual_mean[key] = pd.DataFrame(df[vic_var].
+                                                            groupby(df.index.
+                                                            hour).mean())
```

Comment:
see comments my other comments on line breaking. Breaking after a `.` should be your last resort.